docs/deployment.md

```diff
@@ -52,21 +52,33 @@ kubectl apply -f https://github.com/${KWOK_REPO}/raw/main/kustomize/stage/pod/ch
 kubectl apply -f charts/overrides/kwok/pod-complete.yml
 ```
 
-For configuring virtual nodes, you need to provide the `values.yaml` file to define the type and quantity of nodes you wish to create. You also have the option to enhance node configurations by adding annotations, labels, and conditions. For guidance, refer to the [values-example.yaml](../charts/virtual-nodes/values-example.yaml) file.
+## Setting up virtual nodes
 
-Currently, the system includes the following node types:
-- [dgxa100.40g](https://docs.nvidia.com/dgx/dgxa100-user-guide/introduction-to-dgxa100.html#hardware-overview)
+There are two ways to set up virtual nodes in the cluster:
+
+### 1. Using the `helm` command
+
+Run the `helm install` command and provide the `values.yaml` file that specifies the types and quantities of nodes you wish to create. For example, see the [values-example.yaml](../charts/virtual-nodes/values-example.yaml) file.
+This example includes configurations for the following node types:
 - [dgxa100.80g](https://docs.nvidia.com/dgx/dgxa100-user-guide/introduction-to-dgxa100.html#hardware-overview)
 - [dgxh100.80g](https://docs.nvidia.com/dgx/dgxh100-user-guide/introduction-to-dgxh100.html#hardware-overview)
 - cpu.x86
 
-If you need to introduce additional node types, update the values file used for node configuration with the node information (such as type and count), and include a parameters section in the [nodes.yaml](../charts/virtual-nodes/templates/nodes.yaml) file.
-
-To deploy the nodes in `values-example.yaml`, use the Helm command:
+To deploy the nodes defined in `values-example.yaml`, use the following command:
 ```bash
 helm install virtual-nodes charts/virtual-nodes -f charts/virtual-nodes/values-example.yaml
 ```
 
+### 2. Using the Task Specification
+
+Set up virtual nodes within the `Configure` task in the task specification file. For this example, refer to [test-custom-resource.yml](../resources/tests/test-custom-resource.yml#L11-L19).
```

Comment:
To use this, the user will need to have helm CLI installed on their machine, right? Should we add a blurb about this? Users may assume that for using the configure command, helm does not need to be installed on their system.

Reply:
correct, will add. But that's also true for the first method.

Reply:
Right. But I was worried that people would assume that since the first option needs helm, the second option would now.

Reply:
agree, updated the doc.